### DISCUSSION TITLE
possibility to lock down the source from "off site" links

### DISCUSSION THREAD

**Discussion by Hardcore-fs** [jobrunr/jobrunr#555] [General]
Currently on the integrated web server for jobs,
You have a lot of off site links. , can these  be removed or made into static images.
There is a certain place .. that shall remain nameless, where  DNS requests are "hijacked" to inject material into any web application, using offsite links, for key domains.

Specifically there are a number of central DNS servers that supply injected  malware,  designed to target  a well known companies analytic engine, it's meant to target certain traveling groups who have critical corporate data systems, who rely on external analytical engines to provide web application statistics.

There have been a number of  cases where global web traffic has been redirected via, "mis-configured" global routing  information... (BGP) this is potentially used to re-point DNS requests & inject material into external links in websites.
The system is trialed &  tested internally, then every so often there is an accidental global re-route of traffic, into this preconfigured environment.

**Comment by rdehuyss** [comment]
Currently, i don't see the added benefit to be honest. I could add it on the roadmap of the paying feature but as you're the only one asking for this, it won't get a high prio.

**Comment by Hardcore-fs** [comment]
Hi,
yep.. not ground breaking stuff for most people...(very few people even know what is going on or even consider it)

but it goes towards **security**..., there should be no off site links, in any corporate web app. (yep I know you need to promote it)
also recently for corporate security ratings you get down graded for such things.

I know of several very large branded companies that insist their supplier chains use their web based systems, and since many use  a certain off shore manufacturing center, they have (unknowingly) had their web apps compromised, due to offsite links & browser pre-emptive caching., loading in "poisoned JS", via requisitioned DNS servers.
If these large players actually took the time to get their dev teams to look at these exploits, they would be shocked.

we found one such  DNS "wedge" was actually using JS /code that converted all the HTTPS displayed screen data to a HTTP stream out of the app.(actually meant to target online banking apps.... not job schedulers)
The main problem with staff is they NEVER  have just one damned website logged in at a time.

Admin level staff are most likely to use this screen, so chances are any Azure or cloud sites logged into as admin  are potential targets.
